File: i2f-jdk/i2f-swl/src/main/java/i2f/swl/core/exchanger/impl/SwlEmptyNonceManager.java
package i2f.swl.core.exchanger.impl;

import i2f.swl.core.exchanger.SwlNonceManager;

/**
 * @author Ice2Faith
 * @date 2024/8/4 22:25
 * @desc
 */
public class SwlEmptyNonceManager implements SwlNonceManager {
    @Override
    public boolean contains(String nonce) {
        return false;
    }

    @Override
    public void set(String nonce, long timeoutSeconds) {

    }
}

File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-swl/core/core/exchanger/impl/SwlEmptyNonceManager.js
import SwlNonceManager from "../SwlNonceManager";

/**
 * @return {SwlEmptyNonceManager}
 * @implements {SwlNonceManager}
 * @constructor {SwlEmptyNonceManager}
 */
function SwlEmptyNonceManager() {

}

// 继承
SwlEmptyNonceManager.prototype = Object.create(SwlNonceManager.prototype)
SwlEmptyNonceManager.prototype.constructor = SwlEmptyNonceManager

/**
 * @param nonce {String}
 * @return {boolean}
 */
SwlEmptyNonceManager.prototype.contains = function (nonce) {
    return false
}
/**
 * @param nonce {SwlCert}
 * @param timeoutSeconds {long}
 * @return {void}
 */
SwlEmptyNonceManager.prototype.set = function (nonce,timeoutSeconds) {

}

export default SwlEmptyNonceManager
